### ISSUE TITLE
[2021 fall] CVPR 2021 Boosting Monocular Depth Estimation Models to High-Resolution via Content-Adaptive Multi-Resolution Merging (20185586)

### ISSUE THREAD

**Issue by psh01087** [awesome-davian/awesome-reviews-kaist#147]

**Comment by earlgrey97** [comment]
Hi. I'm Nahyeon, EE student on master's course.

I read your review and here are some minor points that can be fixed.
1) First of all, the figures of your articles are not shown... Please look at the links of your figures and make them be shown...!
2) In part 2.b, the equations are not shown appropriately. For example, it is shown like $R_0$. I think these parts should be fixed.
3) I think you need more explanation on experimental setup part. As I saw the corresponding paper, there are quite a lot of explanation on the method part. I think you can write some more for readers to understand this paper better.

Best regards, 
Nahyeon

**Comment by quangbk2010** [comment]
Hello, this is Quang.
It's my pleasure to comment on your paper review.

Overall, I think this review should be more carefully rewritten to help readers easier to follow.
Besides what Nahyeon commented, there are some minor adjustment maybe considered.

1. The problem definition is not clear.
2. The use of headline should be fixed. For instance: 'Preliminaries' should be smaller; 2.a, 2.b should be 2nd level. 
3. The reference is hard to follow. Maybe we should use text instead.

If there is any misunderstanding, please correct me.
Thank you.

**Comment by madin162** [comment]
Hi, this is Muhammad Adi.
I have read your article and majority of my feed backs have been covered by Nahyeon and Quang, so I wont repeat the similar ones.
Thus, asides from what have written, here are some suggestions that maybe can help:

1. More details can be added to the explanation of the method, for example the overall pipeline of the method and explanation for patch estimates for local boosting. 
2. After the quantitative result image has been correctly displayed, maybe you can also add more explanation to the details or improvements that have been made by the proposed method compared to the previous ones.

Hope the suggestion helps.
Thank you.
